src/main/java/Main.java

```diff
@@ -1,13 +1,25 @@
-import java.text.Normalizer;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Main {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
         ReadFromJSON data = new ReadFromJSON();
         data.readPoints("./JSON_files/Data.json");
 
+        List<Line> lines = new ArrayList<>();
+
+        for (int i = 0; i < data.getInfo().getRoofs().size(); i++) {
+            Line L = new Line(data.getInfo().getRoofs().get(i).getId(),
```

Comment:
would it be possible to put the repeating part in a single variable? example:
`someVariable = data.getInfo().getRoofs().get(i)
 `
and then use it as:
`someVariable.getId(), someVariable.getType()`, etc

Reply:
It is possible and it would improve code readability. However, not exactly like described. If we save every roof to different variable it wouldn't be efficient. And in this particular example we would still need the .get(i) method, as we need to retrieve multiple roofs in the loop. I think if we would change this, first we need to declare new list
`List<Roof> someVariable = data.getInfo().getRoofs();`
and then it would be
`someVariable.get(i).getId() or someVariable.get(i).getType()`
rather than 
`someVariable.getId() or someVariable.getType()`